src/Button.jsx

```diff
@@ -1,8 +1,8 @@
 import { func, number } from 'prop-types';
 import React from 'react';
 
-const Button = ({ count, handleClickBtn }) => (
-  <button type="button" onClick={handleClickBtn}>
+const Button = ({ count, handleClick }) => (
```

Comment:
```suggestion
export default const Button = ({ count, handleClick }) => (
```

따로 아래에서 `export` 하지 않아도 위와 같은 형태로 사용하실 수 있어요 ;)

저는 `export` 하는 이 파일의 메인 컴포넌트도 어떤건지 바로 알수있어서 코드 가독성을 높이는데도 도움이 된다고 생각해요.

Reply:
감사합니다 수정해보겠습니다.